Content.Server/Animals/Components/UdderComponent.cs

```diff
@@ -26,12 +26,6 @@ internal sealed partial class UdderComponent : Component
         [DataField, ViewVariables(VVAccess.ReadOnly)]
         public string SolutionName = "udder";
 
-        /// <summary>
-        ///     The solution to add reagent to.
-        /// </summary>
-        [DataField]
-        public Entity<SolutionComponent>? Solution = null;
```

Comment:
The fields themselves are fine, and should stay there to make retrieving solutions faster. you only need to get rid of the DataField attribute